src/ui/public/utils/lodash-mixins/string.js

```diff
@@ -10,7 +10,7 @@ export default function (_) {
      * @return {string}
      */
     asPrettyString: function (val) {
-      if (val === null || val === undefined) return ' - ';
+      if (val === null || val === undefined) return '(empty)';
```

Comment:
@spalger I've made this change per @jimmyjones2's earlier comment but I don't know the ideal way to style it as gray/italics. Do I simply wrap the "(empty") in a `<span>` with a `class` right here in the string literal? Or is there a better way to do this?